### PR TITLE
v1.42 Mobile Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/mobile-changelog.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
+- Release Date: April 16, 2021
+- Server Versions Supported: Server v5.25+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - **Upgrade to server version v5.31 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.25 has ended and upgrading to server ESR v5.31 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Improvements
+ - Apps Framework
```
Comment (amyblais) [inline source/administration/mobile-changelog.md:13]: Needs a better description, as well as any notes on compatibility / breaking changes (if any). @larkox

File: source/administration/mobile-changelog.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.42.0 Release
+- Release Date: April 16, 2021
+- Server Versions Supported: Server v5.25+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
```
Comment (enahum) [inline source/administration/mobile-changelog.md:5]:
```suggestion
- Server Versions Supported: Server v5.31+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
```

File: source/administration/mobile-changelog.md (hunk 1 above)
Comment (enahum) [reply source/administration/mobile-changelog.md:13]: Shouldn't we be asking better description to the PM instead?

Also this should be highlighted somehow, I do believe this is massive!

File: source/administration/mobile-changelog.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+
+### Compatibility
+ - **Upgrade to server version v5.31 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.25 has ended and upgrading to server ESR v5.31 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Improvements
+ - Apps Framework
+ - Improved performance of scrolling the post list on Android.
```
Comment (enahum) [inline source/administration/mobile-changelog.md:14]: Remove this one please

File: source/administration/mobile-changelog.md [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
+ - Added support for replacing double-dashes with emdashes on iOS.
```
Comment (enahum) [inline source/administration/mobile-changelog.md:15]: Remove this one too

File: source/administration/mobile-changelog.md [hunk 5]
```diff
@@ -1,5 +1,39 @@
...
+ - Fixed an issue where the Mention Highlight Link Color was not being used properly.
+ - Fixed a minor typo in the search **in:** modifier label.
+ - Fixed an issue where OpenGraph without images did not show the image placeholder.
+
+#### Android specific
+ - Fixed an issue with frequent logouts on the latest Android OS.
+ - Fixed an issue where the app crashed when pressing the back button while sharing an image.
+ - Fixed an issue where uploading and sharing PDFs crashed the app.
+ - Fixed an issue where the image gallery view had a weird behaviour and did not close smoothly.
+ - Fixed an issue where sliding did not work when 5 images were uploaded.
```
Comment (enahum) [inline source/administration/mobile-changelog.md:31]: This is for both ios & android

**Review by enahum — CHANGES_REQUESTED** [review]

File: source/administration/mobile-changelog.md (hunk 1 above)
Comment (larkox) [reply source/administration/mobile-changelog.md:13]: @aaronrothschild would be the right person to give a good description here.

Regarding compatibility and breaking changes, it should include none.

File: source/administration/mobile-changelog.md [hunk 6]
```diff
@@ -1,5 +1,39 @@
...
+- Release Date: April 16, 2021
+- Server Versions Supported: Server v5.25+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - **Upgrade to server version v5.31 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.25 has ended and upgrading to server ESR v5.31 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Improvements
+ - Apps Framework
```
Comment (aaronrothschild) [inline source/administration/mobile-changelog.md:13]:
```suggestion
 - Support for the new Apps Framework (Developer Preview)
```

Comment (aaronrothschild) [reply source/administration/mobile-changelog.md:13]: Keep in mind: This feature is not usable on Mobile until the App Framework actually becomes available on webapp and the proxy plugin is loaded on an instance.  That's why I said "support".  The launch for the developer preview of the Apps Framework is at the end of this month (April 29th).

**Review by aaronrothschild — APPROVED** [review]

File: source/administration/mobile-changelog.md (hunk 1 above)
Comment (larkox) [reply source/administration/mobile-changelog.md:13]: Also, compatibility wise, it depends on a feature flag added on on this PR: https://github.com/mattermost/mattermost-server/pull/16851
According to the milestone on the pr, it is server version 5.35.

**Review by enahum — APPROVED** [review]

**Review by aaronrothschild — APPROVED** [review]

**Review by esethna — APPROVED** [review]

**Review by larkox — APPROVED** [review]

**Review by anurag6713 — APPROVED** [review]

**Pull request merged**